Makefile

```diff
@@ -143,12 +145,12 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 .PHONY: fmt
-fmt: ## Run go fmt against code.
-	go fmt ./...
+fmt: goimports ## Run go goimports against code - goimports = go fmt + fixing imports.
+	$(GOIMPORTS) -w  ./main.go ./api ./controllers ./test
```

Comment:
`./pkg` is missing. Also there is no `./test` in this repo (yet), not sure if goimports comlains about it?

Copy / paste is ok, but always try to understand what the copied code is doing, and if it works in your own project 🙂